tests/wpt/metadata/html/dom/idlharness.https.html.ini

```diff
@@ -2407,7 +2407,7 @@
     expected: FAIL
 
   [HTMLLinkElement interface: attribute referrerPolicy]
-    expected: FAIL
+    expected: PASS
```

Comment:
Thanks for the quick fix! 🙏 

While the test expectation is `PASS`, we can just remove it in the `ini` files!


```suggestion
```


(Same for other PASS tests)

Reply:
Ah, updated!